src/inausoft.netCLI/Deserialization/DeserializationException.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace inausoft.netCLI.Deserialization
+{
+    public class DeserializationException : Exception
+    {
+        public ErrorCode? ErrorCode { get; }
```

Comment:
This prop is nullable - why? Can I have a `DeserializationException ` without error code? How should I interpret this kind of error?

Reply:
Adjusted to not nullable.